packages/app/src/specs/SpecsListBanners.vue

```diff
@@ -30,7 +30,7 @@
     </p>
   </Alert>
   <Alert
-    v-if="showFetchError"
+    v-else-if="showFetchError"
     v-model="showFetchError"
```

Comment:
Hmm. The combination of `v-if` and `v-model` pointing to the same value seems like a hint that `v-model` is not doing what we want and hiding/showing the Alert on its own?

Reply:
This was an attempt to simplify the "only show one banner" concern within `SpecsListBanners.vue`. Currently, `showFetchError` just cares about whether the conditions for its display are met independently, but if we remove the `v-if/v-else-if` then I would need to add conditions to check whether any higher-priority banners are shown when determining the model value. This felt like a cleaner approach

Reply:
Yeah that makes sense, at least for this iteration. I know more is coming 👍